testing/RedUNIT/Base/Aliasing.php

```diff
@@ -67,15 +68,15 @@ public function testAutoResolver()
 		//Also test the default implementation...
 		$nullWriter = new \NullWriter( R::getDatabaseAdapter() );
 		asrt( is_null( $nullWriter->inferFetchType( 'test', 'test' ) ), TRUE );
-		//Realteacher should take precedence over fetchAs-teacher, name conventions first!
-		//also: ensure we do not use autoresolv for anything except when truly necessary! (performance)
+		//Realteacher should NOT take precedence over fetchAs-teacher, as there's already a foreign key
```

Comment:
Okay but this breaks BC.

Reply:
However, it is probably safe to assume nobody was using Auto Resolve in frozen mode.

Reply:
Well since it was broken, it's pretty safe to assume I suppose.